src/stock-price-changes/stock-price-changes.routes.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+(function() {
+    'use strict';
+
+    angular
+        .module('stock-price-changes')
+        .config(routes);
+
+    routes.$inject = ['$stateProvider'];
+
+    function routes($stateProvider) {
+        $stateProvider.state('openlmis.stockmanagement.stockPriceChanges', {
+            isOffline: true,
+            url: '/stockPriceChanges'
+                + '?facility&program&supervised&page&size&includeInactive'
+                + '&productCode&productName&lotCode',
+            label: 'stockPriceChanges.title',
+            priority: 1,
+            showInNavigation: true,
+            views: {
+                '@openlmis': {
+                    controller: 'StockPriceChangesController',
+                    controllerAs: 'vm',
+                    templateUrl: 'stock-price-changes/stock-price-changes.html'
+                }
+            },
+            // accessRights: [STOCKMANAGEMENT_RIGHTS.STOCK_ADJUST]
```

Comment:
Should it be commented out? If access rights are needed this comment can be removed

Reply:
yes, fixed the right and now it's not commented.